app/policies/podcast_policy.rb

```diff
@@ -1,4 +1,8 @@
 class PodcastPolicy < ApplicationPolicy
+  def new?
+    token&.authorized_account_ids(:podcast_create).present?
```

Comment:
The `create?` method requires the Podcast have an account set on it already.  So needed a check for "can you create a podcast for _any_ account ids?".